WebApp/webapp/src/components/NavBar/index.module.css

```diff
@@ -0,0 +1,41 @@
+.collapsedOverlay {
+    transition: opacity 0.3s ease;
+  }
+  
+  .expandedOverlay {
+    transition: opacity 0.3s ease;
+  }
+  
+  .companyInfo {
+    display: flex;
+    align-items: center;
+    padding: 1rem;
+    gap: 1rem;
+    color: #dddddd;
```

Comment:
can you change this to the global variables used for the css colors ie --var(--header-color) etc. found in the app.css file.
Any colours used in css files should be written there so theme colours aren't broken